packages/sveltekit/src/vite/autoInstrument.ts

```diff
@@ -101,22 +103,80 @@ export async function canWrapLoad(id: string, debug: boolean): Promise<boolean>
 
   const code = (await fs.promises.readFile(id, 'utf8')).toString();
 
-  const mod = parseModule(code);
+  // Taken from recast's typescript parser config, minus the JSX plugin
```

Comment:
if we're vendoring this in, we need to include the license. Would make sense to extract it to another file then.

Reply:
It's not a 1:1 copy but I agree, close enough to include the license and some more information. Thanks for the suggestion! [c07db78](https://github.com/getsentry/sentry-javascript/pull/15578/commits/c07db7869cb6132a0589a0644f81c742056dfecc)